### ISSUE TITLE
How to use ? Is it Functional?

### ISSUE THREAD

**Issue by ThePanchalShadow** [Nohet/linkvertise.py#3]
Hey bro I have been looking for this for a long I just so your code and there is no proper instruction on how to use it and I would like to know is it working or not properly I really want this wrapper and let's try to figure it out together if you haven't yet

**Comment by Nohet** [comment]
Hi!
There is an example usage available on github with comments that shows you
how to use this library.

**Comment by ThePanchalShadow** [comment]
Bro, I am not able to find the repo you are talking about and see I am able to generate links but they are not showing on the dashboard of my Linkvertise account and I wouldn't know if anyone clicked on this link or not or what is revenue I am generating

and can you share any of your social media so we can talk one on one if possible and thank you for your time mate seriously thank you so much

**Comment by askfriends** [comment]
Im also looking for this wrapper, but it seems to be old repo and not updated, since linkvertise has updated their code recently and introduced some ad styles to choose from when creating a new link, like user will see 1 to 5 counter only, or they will be offered some app to install, all these settings can be applied while creating a new link. So kindly update the library to make it work with the latest linkvertise code.
Thanks

**Comment by ThePanchalShadow** [comment]
> 

Bro the issue is not that link is not generating it generates the link but it doesn't show in my dashboard

**Comment by HipShot-Critical** [comment]
> > 
> 
> Bro the issue is not that link is not generating it generates the link but it doesn't show in my dashboard

I am having this same issue, did linkvertise update the api so this method no longer works? Dose anyone have an update code or work around to have the links appear on users profile?

**Comment by ThePanchalShadow** [comment]
No man not at all

**Comment by Nohet** [comment]
> > > 
> > 
> > 
> > Bro the issue is not that link is not generating it generates the link but it doesn't show in my dashboard
> 
> I am having this same issue, did linkvertise update the api so this method no longer works? Dose anyone have an update code or work around to have the links appear on users profile?

Hi, as far as I can remember, the links never showed up on the profile, but they are working. You can test that by going through the link and seeing that it gave you money.

**Comment by ThePanchalShadow** [comment]
Ok, I'll check it out.

**Comment by HipShot-Critical** [comment]
> s far as I can remember, the links never showed up on the profile, but they are working. You can test that by going through the link and seeing that it gave you mon

They are only working in the sense that they are clickable and go to the desired url. However I don't know if there is a way to conform if the links are actually monetized without having the data show up on the dashboard. I have contacted the support people to try to get some clarification but have have so far gotten pretty poor response back. Sending me a link to the video that shows how you set up the api in wordpress dose not actually help at all for the questions I was asking. Very disappointing to say the least.

**Comment by ThePanchalShadow** [comment]
Yeah that is true it is a really issue I faced too and I'm not currently
using linkverties or any other link shorter so not sure if they improved
